examples/latent_sde.py

```diff
@@ -71,23 +71,47 @@ def f(self, t, y):  # Approximate posterior drift.
         inp = torch.cat((torch.sin(t), torch.cos(t), y), dim=-1)
         return self.net(inp)
 
+    def f_aug(self, t, y):
+        y = y[:, 0][:, None]
+        res = self.f(t, y)
+        z = res - self.h(t, y)
+        g = self.g(t, y)
+        g = torch.where(
+            g.abs() > 1e-7,
+            g,
+            torch.ones_like(g).fill_(1e-7) * g.sign()
+        )
+        z = z / g
+        z = .5 * (torch.norm(z, dim=1) ** 2.)[:, None]
```

Comment:
Use the `keepdim` option of `torch.norm`, instead of explicit dimension unsqueezing.

Reply:
Done!